Add missing step under adding .onmicrosoft domain (#8410)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/setup/add-domain.md

1. In the Microsoft 365 admin center, select **Settings**, and then select **Domains**.

> 1. Click on an existing *.onmicrosoft.com* domain.

Reviewer: ```suggestion
1. Select an existing *.onmicrosoft.com* domain.
```

1. On the **Overview** tab, select **Add onmicrosoft.com domain**.

    ![Domains page.](../../media/onmicrosoft-domains.png)
  

> 1. In the **Add onmicrosoft domain** page, in the **Domain name** box, type the name for your new onmicrosoft.com domain. 

Reviewer: ```suggestion
1. On the **Add onmicrosoft domain** page, in the **Domain name** box, enter the name for your new onmicrosoft.com domain. 
```

1. Go to the Microsoft admin center, **Settings** > <a href="https://go.microsoft.com/fwlink/p/?linkid=834818" target="_blank">**Domains**</a>.

> 2. Click on an existing *.onmicrosoft.com* domain.

Reviewer 2: ```suggestion
2. Select an existing *.onmicrosoft.com* domain.
```
